src/test/java/org/jpeek/skeleton/XmlClassTest.java

```diff
@@ -65,6 +66,18 @@ public void parsesMethodVisibility() {
         );
     }
 
+    @Test
+    public void thereIsNoAttributePublic() {
+        MatcherAssert.assertThat(
+            XhtmlMatchers.xhtml(
+                this.classAsXml("ClassWithDifferentMethodVisibilities")
```

Comment:
@GnusinPavel please, leave a puzzle to extract the method `classAsXml` as a new class. @paulodamaso WDYT?

Reply:
put puzzle